libs/books/feature/src/lib/book-search/book-search.component.scss

```diff
@@ -12,6 +12,9 @@ form {
   display: grid;
   grid-template-columns: 50% 50%;
   grid-template-rows: auto;
+  @media only screen and (max-width: 762px) {
```

Comment:
Change this max-width to 767px for considering mobile views

Reply:
Done